apps/local_laplacian/local_laplacian_generator.cpp

```diff
@@ -124,17 +117,16 @@ class LocalLaplacian : public Halide::Generator<LocalLaplacian> {
             }
             outGPyramid[0]
                 .compute_at(output, y).vectorize(x, 8);
-            for (int j = 5; j < J; j++) {
+            for (int j = 5; j < pyramid_levels; j++) {
                 inGPyramid[j].compute_root();
                 gPyramid[j].compute_root().parallel(k);
                 outGPyramid[j].compute_root();
             }
         }
-
-        return output;
     }
 private:
     Var x, y, c, k;
+    Func remap, gray, inGPyramid[maxJ], gPyramid[maxJ], outGPyramid[maxJ];
```

Comment:
> // foo.specialize(vector_lanes > 1).vectorize(x, vector_lanes);

What does vectorize(1) [or vectorize(0)] do? Is it equivalent to omitting a call to vectorize? Would it "undo" a previous call to vectorize()? If not, should it?

Similarly: parallel(1), etc?